python/ray/data/tests/test_dataset_formats.py

```diff
@@ -446,10 +446,16 @@ def test_parquet_read_bulk(ray_start_regular_shared, fs, data_path):
     path2 = os.path.join(setup_data_path, "test2.parquet")
     pq.write_table(table, path2, filesystem=fs)
 
-    # Expect directory path expansion to fail.
-    with pytest.raises(OSError):
+    # Expect directory path expansion to fail due to default format-based path
+    # filtering.
```

Comment:
Could you add a sentence describing what you mean by format-based path filtering?